x-pack/test/cases_api_integration/security_and_spaces/tests/common/metrics/get_case_metrics.ts

```diff
@@ -17,46 +23,108 @@ export default ({ getService }: FtrProviderContext): void => {
   const kibanaServer = getService('kibanaServer');
 
   describe('case metrics', () => {
-    const closedCaseId = 'e49ad6e0-cf9d-11eb-a603-13e7747d215z';
+    describe('closed case from kbn archive', () => {
```

Comment:
Just reorganized the tests so that those accessing the kbn archive are within a single `describe` block.